myapp/src/day4/Question6/Todo.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react'
+import './styles.css'
+const Todo = () => {
+  const [tasks, setTasks] = useState([])
+  const [value, setValue] = useState('')
+  const handleTask = () => {
+    const newTasks = [...tasks, value]
+    setTasks([...newTasks])
+    setValue('')
+  }
+  const removetask = (index) => {
+    setTasks((prevTask) => prevTask.filter((_, i) => i !== index))
+  }
+  return (
+    <div className="ques6">
+      <div className="inputarea">
+        <input
+          type="text"
+          placeholder="add a new task"
+          value={value}
+          onChange={(e) => setValue(e.target.value)}
+        />
+        <button onClick={handleTask}>Add</button>
```

Comment:
check for empty values before adding to list

Reply:
okay, i had added if condition for that